MC/config/PWGHF/external/generator/generator_pythia8_gaptriggered_hf.C

```diff
@@ -88,54 +141,60 @@ protected:
     return true;
   }
 
-  bool selectEvent(const Pythia8::Event& event)
+  bool selectEvent(const Pythia8::Event &event)
   {
-    bool isGoodAtPartonLevel;
-    bool isGoodAtHadronLevel;
-    if(mDoNoQuarkTrigger){
```

Comment:
You can remove `mDoNoQuarkTrigger` from the class since this was the only time it was used in the code